chart/templates/_helpers.tpl

```diff
@@ -54,9 +54,5 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 Create the name of the service account to use
 */}}
 {{- define "druid-operator.serviceAccountName" -}}
-{{- if .Values.serviceAccount.create }}
 {{- default (include "druid-operator.fullname" .) .Values.serviceAccount.name }}
-{{- else }}
-{{- default "default" .Values.serviceAccount.name }}
-{{- end }}
 {{- end }}
```

Comment:
I prefer to let `druid-operator.fullname`and remove this part

Reply:
Fixed